prometheus-pusher/main.go

```diff
@@ -154,6 +157,10 @@ func (p *prometheusSink) createPusher(jobName string) (*push.Pusher, error) {
 	return pusher, nil
 }
 
+func parseStringToSlice(envValue string) []string {
+	return strings.Split(envValue, ",")
```

Comment:
Will envValue be an empty string  in most cases? In that case, will the split() return an array of one element? Would return nil a better choice - performant later on?

Reply:
It returns an empty array len=0. we are good. but add check